Directory.Build.props

```diff
@@ -25,7 +25,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nerdbank.GitVersioning" Version="3.3.37" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="3.6.133">
```

Comment:
I think we can keep the old default form and avoid to specify PrivateAssets,IncludeAssets in the inner element

Reply:
Done.